range_test.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+import range
+from unittest.mock import patch 
+
+
+class Test(unittest.TestCase):
+  def test_simple(self):
```

Comment:
Test cases need to be strengthened. Add test data with variations for ex. repeating numbers. {3,3,4,5}